specs/glossary.md

```diff
@@ -483,6 +483,8 @@ When submitted to Ethereum calldata, the batcher transaction's receiver must be
 transaction must also be signed by a recognized batch submitter account. The recognized batch submitter account
 is stored in the [System Configuration][system-config].
 
+TODO: clarify batch submission frequency.
+
```

Comment:
```suggestion
## Batch submission frequency

Within the [sequencing-window] constraints the batcher is free to submit L2 blocks to for data-availability at any time. The batcher may be reconfigured with a new policy at any time by its operator. The rollup enforces safety guarantees and liveness through the sequencing window, if the batcher does not submit data within this allotted time.

By submitting new L2 data in smaller more frequent steps, there is less delay in confirmation of the L2 block inputs. This allows verifiers to ensure safety of L2 blocks sooner. This also reduces the time to finality of the data on L1, and thus the time to L2 input-finality.
 
By submitting new L2 data in larger less frequent steps, there is more time to aggregate more L2 data, and thus reduce fixed overhead of the batch-submission work. This can reduce batch-submission costs, especially for lower throughput chains that do not fill data-transactions (typically 128 KB of calldata, or 800 KB of blobdata) as quickly.
```

Reply:
Needs TOC update.